omnigibson/object_states/particle_modifier.py

```diff
@@ -274,34 +273,6 @@ def overlap_callback(hit):
                 orientation=T.euler2quat([0, -np.pi / 2, 0]),
             )
 
-            # Generate the projection visualization
```

Comment:
Hmmmm...this is gonna cause some serious merge conflicts with the PR I also made. Was the reason to move this to ParticleApplier to avoid creating it for the ParticleRemover? I kept it as part of ParticleModifier but added a "visualize" flag so that it won't necessarily be rendered, thereby providing flexibility for all downstream classes but keeping it as a core feature that can be leveraged. what do you think?

Reply:
I am not sure why we need to create visualization if it's invisible. But let's discuss in person.

Reply:
I will move it back the original location with a `pass` for ParticleRemover for future implementation